src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.RAND.cs

```diff
@@ -9,16 +9,15 @@ internal static partial class Interop
 {
     internal static partial class Crypto
     {
-        internal static unsafe bool GetRandomBytes(byte* pbBuffer, int count)
+        internal static bool GetRandomBytes(ref byte pbBuffer, int count)
         {
-            Debug.Assert(pbBuffer != null);
```

Comment:
> a suggestion that ref byte instead of byte* could eliminate an unsafe from a place passing null

With unsafe you can do almost anything.  We would also need ptrs and unsafe code to write such an assert.
```C#
using System.Diagnostics;

class Program
{
    static unsafe void Main()
    {
        byte* b = null;
        Bar(ref *b);
    }

    static unsafe void Bar(ref byte b)
    {
        fixed (byte* bPtr = &b) Debug.Assert(bPtr != null);
    }
}
```